ddocs/medic-db/medic/nouveau/docs_by_replication_key/default_analyzer

```diff
@@ -0,0 +1 @@
+whitespace
```

Comment:
I feel like the default analyzer could just be keyword. I don't think we will ever need to be tokenized by whitespace, right?